mcp_run_python/deno/src/main.ts

```diff
@@ -1,47 +1,52 @@
 // deno-lint-ignore-file no-explicit-any
 /// <reference types="npm:@types/node@22.12.0" />
 
-import './polyfill.ts'
-import http from 'node:http'
-import { randomUUID } from 'node:crypto'
-import { parseArgs } from '@std/cli/parse-args'
-import { StdioServerTransport } from '@modelcontextprotocol/sdk/server/stdio.js'
-import { StreamableHTTPServerTransport } from '@modelcontextprotocol/sdk/server/streamableHttp.js'
-import { isInitializeRequest } from '@modelcontextprotocol/sdk/types.js'
-import { type LoggingLevel, SetLevelRequestSchema } from '@modelcontextprotocol/sdk/types.js'
-import { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js'
-import { z } from 'zod'
-
-import { asJson, asXml, RunCode } from './runCode.ts'
-import { Buffer } from 'node:buffer'
-
-const VERSION = '0.0.13'
+import './polyfill.ts';
+import http from 'node:http';
+import { randomUUID } from 'node:crypto';
+import { parseArgs } from '@std/cli/parse-args';
+import { StdioServerTransport } from '@modelcontextprotocol/sdk/server/stdio.js';
+import { StreamableHTTPServerTransport } from '@modelcontextprotocol/sdk/server/streamableHttp.js';
+import { isInitializeRequest } from '@modelcontextprotocol/sdk/types.js';
+import {
+  type LoggingLevel,
+  SetLevelRequestSchema,
+} from '@modelcontextprotocol/sdk/types.js';
+import { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js';
+import { z } from 'zod';
+
+import { asJson, asXml, RunCode } from './runCode.ts';
+import { Buffer } from 'node:buffer';
+
+const VERSION = '0.0.13';
 
 export async function main() {
-  const { args } = Deno
+  const { args } = Deno;
   const flags = parseArgs(Deno.args, {
-    string: ['deps', 'return-mode', 'port'],
-    default: { port: '3001', 'return-mode': 'xml' },
-  })
-  const deps = flags.deps?.split(',') ?? []
+    string: ['deps', 'return-mode', 'port', 'host'],
+    default: { port: '3001', 'return-mode': 'xml', host: '127.0.0.1' },
+  });
+  const deps = flags.deps?.split(',') ?? [];
   if (args.length >= 1) {
     if (args[0] === 'stdio') {
-      await runStdio(deps, flags['return-mode'])
-      return
+      await runStdio(deps, flags['return-mode']);
```

Comment:
We've got a bunch more linter edits here, including the import sorting up top. can you please revert that so the PR only has the necessary changes?